WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
@barryceelen Let's see what the others say about this. /cc @JDGrimes @westonruter @GaryJones 

There are some instances in WPCS where we have deprecated properties before (not whitelist comments AFAIK) and those generally do throw a warning.
For an example, see: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/NamingConventions/ValidVariableNameSniff.php#L313